clang/test/Driver/Inputs/sycl.hpp

```diff
@@ -0,0 +1,674 @@
+#pragma once
```

Comment:
Hi @AlexeySachkov 

I am expecting to add more tests to verify that tools work as expected. This does not seem to be a major overhead. I do agree that this creates some maintenance issues. But I expect this to be useful.

Thanks

Reply:
Done. Thanks for comments.